contracts/FlowCallbackScheduler.cdc

```diff
@@ -375,24 +362,21 @@ access(all) contract FlowCallbackScheduler {
             priority: Priority,
             executionEffort: UInt64
         ): EstimatedCallback? {
```

Comment:
this shouldn't have to be an optional anymore since EstimateCallback now has error field